make/common/NativeCompilation.gmk

```diff
@@ -473,68 +477,61 @@ define SetupNativeCompilation
                 #
                 # $(OBJCOPY) --add-gnu-debuglink=... corrupts SUNW_* sections.
                 # Use $(ADD_GNU_DEBUGLINK) until a fixed $(OBJCOPY) is available.
-                $$($1_OBJECT_DIR)/$$(LIBRARY_PREFIX)$$($1_LIBRARY).debuginfo : $$($1_TARGET) \
+              $$($1_DEBUGINFO_FILES): $$($1_TARGET) \
 				$(FIX_EMPTY_SEC_HDR_FLAGS) $(ADD_GNU_DEBUGLINK)
 		$(RM) $$@
 		$(FIX_EMPTY_SEC_HDR_FLAGS) $(LOG_INFO) $$<
 		$(OBJCOPY) --only-keep-debug $$< $$@
 		$(CD) $$(@D) && $(ADD_GNU_DEBUGLINK) $(LOG_INFO) $$(@F) $$<
+			$(TOUCH) $$@
               endif
-            else # not solaris
+          else ifeq ($(OPENJDK_TARGET_OS), linux)
               ifneq ($$($1_STRIP_POLICY), no_strip)
-                $$($1_OBJECT_DIR)/$$(LIBRARY_PREFIX)$$($1_LIBRARY).debuginfo : $$($1_TARGET)
+              $1_DEBUGINFO_FILES := $$($1_OBJECT_DIR)/$$($1_NOSUFFIX).debuginfo
+              $$($1_DEBUGINFO_FILES): $$($1_TARGET)
 		$(RM) $$@
-                ifeq ($(OPENJDK_TARGET_OS), aix)
-		$(CP) -f $$< $$@
-                else
 		$(OBJCOPY) --only-keep-debug $$< $$@
 		$(CD) $$(@D) && $(OBJCOPY) --add-gnu-debuglink=$$(@F) $$<
```

Comment:
Relative to:
* 8025936: Windows .pdb and .map files does not have proper dependencies setup
this line is missing:
```
			$(TOUCH) $$@
```

Reply:
Added it back.